src/commands/account/default.ts

```diff
@@ -32,98 +33,51 @@ export class DefaultAccount extends SwankyCommand<typeof DefaultAccount> {
   async run(): Promise<void> {
     const { args, flags } = await this.parse(DefaultAccount);
 
-    const systemConfig = await getSwankySystemConfig();
+    const configType = flags.global ? "global" : isLocalConfigCheck() ? "local" : "global";
+    const config = configType === "global" ? getSwankyConfig("global") : this.swankyConfig;
```

Comment:
If `configType` is local, we will store `this.swankyConfig` in `config`. But it is not exactly local config, because we have merged accounts from both configs there. When we, for example, pick an account that exists only in the system config, it will change the local config `defaultAccount` field.